aiidalab_widgets_base/display.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import print_function
+
+import importlib
+from IPython.display import display
+
+RECOGNIZED_AIIDA_DATA_VISUALIZERS = {
```

Comment:
Perhaps `AIIDA_VISUALIZER_MAPPING`?

Reply:
Sure. That is definitely a better name.

Thanks